src/web/components/KeyPairs/KeyPairsTable.tsx

```diff
@@ -34,6 +36,9 @@ function KeyPairsTable({
     setShowKeyPairDialog(false);
   };
 
+  const { sortData } = useSortable<KeyPairModel>();
+  const sortedKeyPairs = sortData(keyPairs);
```

Comment:
nit: in some places we would just call this variable `sortedData`. Perhaps we should be consistent? Maybe always name it something useful like we have here? Examples are in `KeyTable.tsx`, `BusinessContactsTable.tsx` and `ParticipantsTable.tsx`

Reply:
I think `sortedKeyPairs` is definitely better than `sortedData` - I don't think it's all that important to update other places though, if the meaning is clear.